sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/services/OrchestrationService.java

```diff
@@ -362,6 +362,34 @@ public OrchestrationChatResponse grounding(
     return client.chatCompletion(prompt, configWithGrounding);
   }
 
+  /**
+   * Using grounding via a sharepoint repository to provide additional context to the AI model.
+   *
+   * @link <a href="https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/grounding">SAP
```

Comment:
(Comment)

Not a fan of putting volatile URLs in JavaDoc, but since this is a test app and already somewhat-established practice in this class - i won't complain further ;)